common/src/main/java/com/revenuecat/purchases/common/caching/DeviceCache.kt

```diff
@@ -110,8 +122,9 @@ open class DeviceCache(
 
     @Synchronized
     fun cacheCustomerInfo(appUserID: String, info: CustomerInfo) {
-        val jsonObject = info.jsonObject.also {
-            it.put("schema_version", PURCHASER_INFO_SCHEMA_VERSION)
+        val jsonObject = info.rawData.also {
+            it.put(CUSTOMER_INFO_SCHEMA_VERSION_KEY, PURCHASER_INFO_SCHEMA_VERSION)
+            it.put(CUSTOMER_INFO_VERIFICATION_RESULT_KEY, info.entitlements.verification.name)
```

Comment:
This is a not ideal consequence of caching `CustomerInfo` but having the verification result property live in `EntitlementInfos`. But I think it's not that bad. Lmk what you think.